report/sections/Chapter 01 - Introduction/Intro.tex

```diff
@@ -1,22 +1,21 @@
-The Haskell programming language is purely functional\cite{} and builds upon the lambda calculus model of computation \cite{}.
+The Haskell programming language is purely functional\cite{Haskell_landing} and builds upon the lambda calculus model of computation \cite{Haskell_book}.
 Unlike many other programming languages, Haskell can both be interpreted or compiled into an executable binary file.
 It is a statically typed language implementing many mathematical constructs such as currying, set-builder notation, recursive definitions of both types and functions, and more.
 This allows the programmer to create concise solutions that would normally be quite verbose in other programming paradigms.
-Therefore, many companies such as Meta and Galois use Haskell for cryptography, avionics monitoring and spam filtering.
+Therefore, many companies such as Meta and Galois use Haskell for cryptography, avionics monitoring and spam filtering\cite{Haskell_companies}.
 Thus, Haskell and other languages within the functional programming paradigm are useful tools for software engineers.
 
 
 Haskell is taught in the Programming Paradigms course on the 1st semester of the Software Master's Degree at \aau{}.
 Most students attending this course failed in the past.
 This resulted in a survey being sent out, and a student task force being assembled to understand the issues that caused the high failure rate.
-Here, many students mentioned not feeling experienced enough with the programming languages used in the course to teach the functional and logical programming paradigms.
+Here, many students mentioned not feeling experienced enough with the programming languages used in the course to teach the functional and logical programming paradigms. This can be seen from the slide shown in appendix \ref{chap:PP-intro}.
```

Comment:
```suggestion
Here, many students mentioned not feeling experienced enough with the programming languages used in the course to teach the functional and logical programming paradigms. This can be in appendix \ref{chap:PP-intro}.
```

Reply:
```suggestion
Here, many students mentioned not feeling experienced enough with the programming languages used in the course to teach the functional and logical programming paradigms. This can be seen in appendix \ref{chap:PP-intro}.
```